rust/agents/validator/Cargo.toml

```diff
@@ -11,8 +11,11 @@ version.workspace = true
 
 [dependencies]
 async-trait.workspace = true
+axum.workspace = true
 config.workspace = true
 derive_more.workspace = true
+derive-new.workspace = true
+reqwest.workspace = true
```

Comment:
this should be under `dev-dependencies` everywhere iiuc 
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3078#discussion_r1432704371